app/src/test/java/io/intrepid/contest/screens/contestcreation/addcategoriestocontest/AddCategoriesPresenterTest.java

```diff
@@ -0,0 +1,37 @@
+package io.intrepid.contest.screens.contestcreation.addcategoriestocontest;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static junit.framework.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AddCategoriesPresenterTest {
+    @Mock
+    AddCategoriesContract.View mockView;
+    AddCategoriesContract.Presenter addCategoryPresenter;
+
+    @Before
+    public void setup(){
+        MockitoAnnotations.initMocks(this);
+        addCategoryPresenter = new AddCategoriesPresenter(mockView, TestPresenterConfiguration.createTestConfiguration());
+    }
+
+    @Test
+    public void presenterCanBeCreated(){
+        assertTrue(addCategoryPresenter != null);
+    }
+    @After
+    public void tearDown(){
+        addCategoryPresenter = null;
```

Comment:
you actually dont need to do this, your setup will get called every time so you dont need to null it out.